shipments_tracker/db/migrate/20230804000126_create_carriers.rb

```diff
@@ -0,0 +1,11 @@
+class CreateCarriers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :carriers do |t|
+      t.string :name,       null: false
+      t.string :url,        null: false
+      t.string :settings
```

Comment:
please update to jsonb field type
```suggestion
      t.jsonb :settings
```

Reply:
Done